### PR TITLE
USAGOV-2101: Add CODEOWNERS

### PR DESCRIPTION
**Pull request by akf**
## Jira Task

https://cm-jira.usa.gov/browse/USAGOV-2101

## Description

Add a CIODEOWNERS file. This initial version identifies users to be notified on PRs that touch benefits-finder code.

## Type of Changes

- [ ] New Feature
- [ ] Bugfix
- [ ] Frontend (Twig, Sass, JS)
  - Add screenshot showing what it should look like
- [ ] Drupal Config (requires "drush cim")
- [ ] New Modules (requires rebuild)
- [ ] Documentation
- [ ] Infrastructure
  - [ ] CMS
  - [ ] WAF
  - [ ] WWW
  - [ ] Egress
  - [ ] Tools
  - [ ] Cron
- [x] Other - git configuration

### Change Requirements

- [ ] Requires New Documentation (Link: {})
- [ ] Requires New Config
- [ ] Requires New Content

### Validation Steps

I created a branch in which I modified the README.md in the usagov_benefit_finder module and made a PR against this branch (I have since closed the PR and deleted the branch). Scott and Gene should have received a notification from GitHub to review the PR.

The other thing to check is -- are the entries in this CODEOWNERS file correct for the purpose of giving Scott and Gene a heads-up when code for the benefit finder tool is updated? 

## Security Review

- [ ] Adds/updates software (including a library or Drupal module)
- [ ] Communication with external service
- [ ] Changes permissions or workflow
- [ ] Requires SSPP updates

## Reviewer Reminders

- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions

Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
2. Find the commit of this pull request.
3. Build and deploy the changes.
4. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.

### CODE REVIEW

File: docs/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# See https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners
+
+# The CODEOWNERS file itself should have eyes on it.
+docs/CODEOWNERS @akf @arpage
+
+# Benefits-finder tool team 
+web/modules/custom/usagov_benefit_finder @scottqueen-bixal @gchi25
+config/sync/*bears* @scottqueen-bixal @gchi25
+config/sync/*benefit_* @scottqueen-bixal @gchi25
+config/sync/*benefits_finder* @scottqueen-bixal @gchi25
```
Comment (scottqueen-bixal) [inline docs/CODEOWNERS:17]:
```suggestion
config/sync/*benefits_finder* @scottqueen-bixal @gchi25
.github/workflows/*benefits-finder* @scottqueen-bixal @XavierMetichecchia 
```

We could consider adding any associated actions

**Review by scottqueen-bixal — CHANGES_REQUESTED** [review]

**Comment by scottqueen-bixal** [comment]
This seems about right to me @akf. 

I'll have @gchi25 to review the config patterns and ensure we are covered there.

**Comment by gchi25** [comment]
Benefit finder has following paragraphs:
Benefit Eligibility (b_benefit_eligibility)
Criteria (b_levent_elg_criteria)
Criteria Group (b_levent_elg_criteria_group)
Relevant Benefit (b_levent_relevant_benefit)
Section (b_levent_elg_section)

The paragraphs and their fields begin with "b_".

$ ls -1 | grep paragraph | grep b_
core.entity_form_display.paragraph.b_benefit_eligibility.default.yml
core.entity_form_display.paragraph.b_levent_elg_criteria.default.yml
core.entity_form_display.paragraph.b_levent_elg_criteria_group.default.yml
core.entity_form_display.paragraph.b_levent_elg_section.default.yml
core.entity_form_display.paragraph.b_levent_relevant_benefit.default.yml
core.entity_view_display.paragraph.b_benefit_eligibility.default.yml
core.entity_view_display.paragraph.b_levent_elg_criteria.default.yml
core.entity_view_display.paragraph.b_levent_elg_criteria_group.default.yml
core.entity_view_display.paragraph.b_levent_elg_section.default.yml
core.entity_view_display.paragraph.b_levent_relevant_benefit.default.yml
field.field.paragraph.b_benefit_eligibility.field_b_acceptable_values.yml
field.field.paragraph.b_benefit_eligibility.field_b_criteria_key.yml
field.field.paragraph.b_benefit_eligibility.field_b_label.yml
field.field.paragraph.b_levent_elg_criteria.field_b_children.yml
field.field.paragraph.b_levent_elg_criteria.field_b_criteria_key.yml
field.field.paragraph.b_levent_elg_criteria.field_b_error_message.yml
field.field.paragraph.b_levent_elg_criteria.field_b_hint.yml
field.field.paragraph.b_levent_elg_criteria.field_b_legend.yml
field.field.paragraph.b_levent_elg_criteria.field_b_required.yml
field.field.paragraph.b_levent_elg_criteria_group.field_b_criterias.yml
field.field.paragraph.b_levent_elg_criteria_group.field_b_description.yml
field.field.paragraph.b_levent_elg_criteria_group.field_b_heading.yml
field.field.paragraph.b_levent_elg_section.field_b_criterias.yml
field.field.paragraph.b_levent_elg_section.field_b_description.yml
field.field.paragraph.b_levent_elg_section.field_b_heading.yml
field.field.paragraph.b_levent_relevant_benefit.field_b_body.yml
field.field.paragraph.b_levent_relevant_benefit.field_b_cta.yml
field.field.paragraph.b_levent_relevant_benefit.field_b_life_event_form.yml
field.field.paragraph.b_levent_relevant_benefit.field_b_link.yml
field.storage.paragraph.field_b_acceptable_values.yml
field.storage.paragraph.field_b_body.yml
field.storage.paragraph.field_b_children.yml
field.storage.paragraph.field_b_criteria_key.yml
field.storage.paragraph.field_b_criterias.yml
field.storage.paragraph.field_b_cta.yml
field.storage.paragraph.field_b_description.yml
field.storage.paragraph.field_b_error_message.yml
field.storage.paragraph.field_b_heading.yml
field.storage.paragraph.field_b_hint.yml
field.storage.paragraph.field_b_label.yml
field.storage.paragraph.field_b_legend.yml
field.storage.paragraph.field_b_life_event_form.yml
field.storage.paragraph.field_b_link.yml
field.storage.paragraph.field_b_required.yml
paragraphs.paragraphs_type.b_benefit_eligibility.yml
paragraphs.paragraphs_type.b_levent_elg_criteria.yml
paragraphs.paragraphs_type.b_levent_elg_criteria_group.yml
paragraphs.paragraphs_type.b_levent_elg_section.yml
paragraphs.paragraphs_type.b_levent_relevant_benefit.yml

**Review by gchi25 — CHANGES_REQUESTED** [review]
Benefit finder has paragraphs:

- Benefit Eligibility (b_benefit_eligibility)
- Criteria (b_levent_elg_criteria)
- Criteria Group (b_levent_elg_criteria_group)
- Relevant Benefit (b_levent_relevant_benefit)
- Section (b_levent_elg_section)

The paragraphs and their fields begin with "b_"
% ls -1 | grep paragraph | grep b_
core.entity_form_display.paragraph.b_benefit_eligibility.default.yml
core.entity_form_display.paragraph.b_levent_elg_criteria.default.yml
core.entity_form_display.paragraph.b_levent_elg_criteria_group.default.yml
core.entity_form_display.paragraph.b_levent_elg_section.default.yml
core.entity_form_display.paragraph.b_levent_relevant_benefit.default.yml
core.entity_view_display.paragraph.b_benefit_eligibility.default.yml
core.entity_view_display.paragraph.b_levent_elg_criteria.default.yml
core.entity_view_display.paragraph.b_levent_elg_criteria_group.default.yml
core.entity_view_display.paragraph.b_levent_elg_section.default.yml
core.entity_view_display.paragraph.b_levent_relevant_benefit.default.yml
field.field.paragraph.b_benefit_eligibility.field_b_acceptable_values.yml
field.field.paragraph.b_benefit_eligibility.field_b_criteria_key.yml
field.field.paragraph.b_benefit_eligibility.field_b_label.yml
field.field.paragraph.b_levent_elg_criteria.field_b_children.yml
field.field.paragraph.b_levent_elg_criteria.field_b_criteria_key.yml
field.field.paragraph.b_levent_elg_criteria.field_b_error_message.yml
field.field.paragraph.b_levent_elg_criteria.field_b_hint.yml
field.field.paragraph.b_levent_elg_criteria.field_b_legend.yml
field.field.paragraph.b_levent_elg_criteria.field_b_required.yml
field.field.paragraph.b_levent_elg_criteria_group.field_b_criterias.yml
field.field.paragraph.b_levent_elg_criteria_group.field_b_description.yml
field.field.paragraph.b_levent_elg_criteria_group.field_b_heading.yml
field.field.paragraph.b_levent_elg_section.field_b_criterias.yml
field.field.paragraph.b_levent_elg_section.field_b_description.yml
field.field.paragraph.b_levent_elg_section.field_b_heading.yml
field.field.paragraph.b_levent_relevant_benefit.field_b_body.yml
field.field.paragraph.b_levent_relevant_benefit.field_b_cta.yml
field.field.paragraph.b_levent_relevant_benefit.field_b_life_event_form.yml
field.field.paragraph.b_levent_relevant_benefit.field_b_link.yml
field.storage.paragraph.field_b_acceptable_values.yml
field.storage.paragraph.field_b_body.yml
field.storage.paragraph.field_b_children.yml
field.storage.paragraph.field_b_criteria_key.yml
field.storage.paragraph.field_b_criterias.yml
field.storage.paragraph.field_b_cta.yml
field.storage.paragraph.field_b_description.yml
field.storage.paragraph.field_b_error_message.yml
field.storage.paragraph.field_b_heading.yml
field.storage.paragraph.field_b_hint.yml
field.storage.paragraph.field_b_label.yml
field.storage.paragraph.field_b_legend.yml
field.storage.paragraph.field_b_life_event_form.yml
field.storage.paragraph.field_b_link.yml
field.storage.paragraph.field_b_required.yml
paragraphs.paragraphs_type.b_benefit_eligibility.yml
paragraphs.paragraphs_type.b_levent_elg_criteria.yml
paragraphs.paragraphs_type.b_levent_elg_criteria_group.yml
paragraphs.paragraphs_type.b_levent_elg_section.yml
paragraphs.paragraphs_type.b_levent_relevant_benefit.yml

File: docs/CODEOWNERS (hunk 1 above)
Comment (gchi25) [inline docs/CODEOWNERS:17]:
```suggestion
config/sync/*benefits_finder* @scottqueen-bixal @gchi25
config/sync/*paragraph*b_* @scottqueen-bixal @gchi25
```

Comment (akf) [reply docs/CODEOWNERS:17]: Added!

**Comment by akf** [comment]
@gchi25 thanks! I added patterns for all those.

**Review by gchi25 — APPROVED** [review]

File: docs/CODEOWNERS (hunk 1 above)
Comment (akf) [reply docs/CODEOWNERS:17]: I didn't mean to ignore this, by the way. I just didn't see it until now!

**Pull request merged**